pages/specification/migration/_index.md

```diff
@@ -3,5 +3,78 @@ title: Migrating from older drafts
 section: docs
 ---
 
-The release notes discuss the changes impacting users and implementers:
+### Introduction
 
+Migrations are like version control for your system, allowing your team to define and share the schema progressions. If you have ever had to figure out when a keyword was introduced and how it behaved, you've faced the problem that migration guides solve.
+
+If you're moving from one Draft to another, learn how to use our migration guides and tooling ([alterschema](https://alterschema.sourcemeta.com/)) to bring your work with you.
+
+### Keywords Overview
+
+Here is a comprehensive overview to get you going.
+
+| All Keywords          | Specification          | Draft introduction | Removed | Changed                       |
```

Comment:
I think there are some issues with the Specification column. We only have "Core" and "Validation" specifications, and those specifications may define multiple vocabularies (like "Applicator" and "Content"), which in turn define keywords of certain types like "Assertion" or "Annotation".

If the intention is to only highlight the specification (which I think it makes sense), then let's stick to only "Core" and "Validation"?

Reply:
I think you might be conflating specification with vocabulary. For example, `contentSchema` is a keyword from the "Content" vocabulary, which is defined in the "Validation" spec

Reply:
Yes, I traced it to the [Specifications](https://json-schema.org/draft/2020-12/json-schema-core#section-10.3.1.1). Thank you.